src/main/java/db/migration/V173__Ensure_Activity_Duration_is_not_zero.java

```diff
@@ -11,50 +11,14 @@
 
     @Override
     public void migrate(Connection connection) throws Exception {
-        // Find all activities
-
-        PreparedStatement psActivities = connection.prepareStatement("SELECT * FROM `Activities`");
-
         try {
-            connection.setAutoCommit(false);
-
-            ResultSet rsActivities = psActivities.executeQuery();
-
-            while (rsActivities.next()) {
-                long activityId = rsActivities.getLong("Id");
-                Time startTime = rsActivities.getTime("StartTime");
-                Time endTime = rsActivities.getTime("EndTime");
-
-                if (startTime == null || endTime == null) {
-                    continue;
-                }
-
-                if (startTime.equals(endTime) ) {
-                    // Add sectionGroup association
-                    PreparedStatement psActivity = connection.prepareStatement(
-                            " DELETE FROM `Activities` WHERE `Id` = ?;"
-                    );
-
-                    psActivity.setLong(1, activityId);
-                    psActivity.execute();
-                    psActivity.close();
-                }
-            }
-
-            // Commit changes
-            connection.commit();
+            PreparedStatement psActivities = connection.prepareStatement("Update Activities SET EndTime = DATE_ADD(EndTime, INTERVAL 50 MINUTE) WHERE StartTime = EndTime");
```

Comment:
What happens if StartTime and EndTime are both Null? Do they equate and this erroneously sets EndTime = Null + 50 minutes?

Reply:
In testing, I actually discovered that
'WHERE StartTime = EndTime' will NOT return rows where StartTime and EndTime are both null.

I'm not sure why it was designed that way, but it let me simplify the query.

Reply:
Interesting. Good to know, glad you beat me to it!